analysis/pricesignal.py

```diff
@@ -24,9 +18,7 @@ def getmef(
 def getaef(
     region, month, basepath=os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 ):
-    """
-    Get the MEF (Marginal emissions factor) for a given region and month.
-    """
+    """Get the MEF (Marginal emissions factor) for a given region and month."""
```

Comment:
This is on me but we should change this comment to be AEF instead of MEF for the getaef function.